ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/cucushift/openshift-e2e-ibmcloud-heterogeneous-cucushift-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: openshift-e2e-ibmcloud-heterogeneous-cucushift
+  steps:
+    pre:
+    - chain: ipi-conf-ibmcloud
+    - ref: ipi-conf-etcd-on-ramfs
+    - chain: ipi-install
+    - ref: upi-install-heterogeneous-powervs
+    - ref: enable-qe-catalogsource
+    - chain: cucushift-installer-check
+    test:
+    - chain: openshift-e2e-test-qe
```

Comment:
BTW, this test chain contains `ginkgo` test, `cucushift` test, and `cypress` test.  Name it as `cucushift` test may confuse the team.

Reply:
OK, I think we can do that....